packages/design-system/src/styles/components/_Badge.scss

```diff
@@ -12,14 +12,23 @@ $badge-font-size-big: $base-font-size;
   /* stylelint-enable scss/media-feature-value-dollar-variable */
 }
 
-@mixin badge-high-contrast-mode-label($label) {
-  /* stylelint-disable scss/media-feature-value-dollar-variable */
+@mixin badge-high-contrast-mode-label($label-en, $label-es: null) {
+  /* stylelint-disable */
   @media (-ms-high-contrast: active), (forced-colors: active) {
-    &::before {
-      content: $label + ': ';
+    [lang^='en'] & {
+      &::before {
+        content: $label-en + ': ';
+      }
+    }
+    @if ($label-es != null) {
+      [lang='es'] & {
```

Comment:
Do we want this to also be `^=` instead of `=`?

Reply:
I was thinking of situations where the consuming design system has something like `lang="en-US"` set up.

Reply:
I figured that, and I think `es` should get the same treatment

Reply:
(See above comments about `es-MX` and `es-US`)